js_modules/dagit/packages/core/src/runs/LogsRow.tsx

```diff
@@ -185,6 +185,8 @@ export const LOGS_ROW_STRUCTURED_FRAGMENT = gql`
       fileKey
       stepKeys
       externalUrl
+      externalStdoutUrl
+      externalStderrUrl
```

Comment:
could also make it a filter on externalUrl with an enum key? like ```externalUrl(STDOUT)``` do you think we'll ever support additional stream types?

Reply:
Do we still need externalUrl?

Reply:
I don't think we need `externalUrl` anymore... you probably want both types because they correspond to the different panel views.

We do need to keep supporting the serialized event metadata field `external_url` though (although, I'm fairly confident that no user OSS or cloud has ever populated it besides me).